packages/components/src/components/lifecycleValidation.spec.ts

```diff
@@ -22,6 +23,9 @@ const tagNamesPublicWithProps = TAG_NAMES.filter(
 const tagNamesPublicWithoutProps = TAG_NAMES.filter(
   (tagName) => !getComponentMeta(tagName).isInternal && !getComponentMeta(tagName).props
 );
+const tagNamesWithBreakpointCustomizableProps = TAG_NAMES.filter(
+  (tagName) => getComponentMeta(tagName).breakpointCustomizableProps
```

Comment:
What about other props that can be objects, like `p-table-head-cell`'s `sort`, `p-scroller`'s `scrollToPosition` and all `intl` and `aria` props?